script-reference/statements.rst

```diff
@@ -456,6 +493,9 @@ Here are the statements that the Zeek scripting language supports.
   * - :zeek:keyword:`add`, :zeek:keyword:`delete`
     - Add or delete elements
 
+  * - :zeek:keyword:`assert`
+    - Runtime sanity checking
```

Comment:
Some consider "sanity check" abelist, maybe go with something like _assertion_ instead. Here and below.

```suggestion
    - Runtime assertion
```

Reply:
I was not aware, thanks.